frontend/src/routes/Governance/discovered/grouping.ts

```diff
@@ -212,8 +226,14 @@ export function grouping(): {
       ) {
         responseAction = 'inform/enforce'
       } else {
-        // Ignore the SonarCloud recommendation of sorting by locale since this is an API field.
-        responseAction = allResponseActionsList.sort().join('/') //NOSONAR
+        if (group[0].kind === 'ValidatingAdmissionPolicyBinding') {
+          responseAction = [...new Set(allResponseActionsList.map((str: string) => str.split('/')).flat())]
```

Comment:
Sorry for missing this before. Instead of this, I think this would be better:

```diff
diff --git a/frontend/src/routes/Governance/discovered/grouping.ts b/frontend/src/routes/Governance/discovered/grouping.ts
index 96e3a0f4..8a3ca79a 100644
--- a/frontend/src/routes/Governance/discovered/grouping.ts
+++ b/frontend/src/routes/Governance/discovered/grouping.ts
@@ -194,7 +194,13 @@ export function grouping(): {
 
         if (responseAction) {
           policy.responseAction = responseAction
-          allResponseActions.add(responseAction)
+          if (policy.kind === 'ValidatingAdmissionPolicyBinding') {
+            for (const action of responseAction.split('/')) {
+              allResponseActions.add(action)
+            }
+          } else {
+            allResponseActions.add(responseAction)
+          }
         }
 
         if (source.type === 'Multiple') {
@@ -226,14 +232,8 @@ export function grouping(): {
       ) {
         responseAction = 'inform/enforce'
       } else {
-        if (group[0].kind === 'ValidatingAdmissionPolicyBinding') {
-          responseAction = [...new Set(allResponseActionsList.map((str: string) => str.split('/')).flat())]
-            .sort() //NOSONAR
-            .join('/')
-        } else {
-          // Ignore the SonarCloud recommendation of sorting by locale since this is an API field.
-          responseAction = allResponseActionsList.sort().join('/') //NOSONAR
-        }
+        // Ignore the SonarCloud recommendation of sorting by locale since this is an API field.
+        responseAction = allResponseActionsList.sort().join('/') //NOSONAR
       }
 
       return {
```